pythoncalculator/multiply.py

```diff
@@ -0,0 +1,2 @@
+def multiply(x, y):
+    return x + y
```

Comment:
Looks like we're using the wrong operator!

```suggestion
    return x * y
```